File: Baekjoon_Online_Judge/NO_13866/13866.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 06                 | #
# | Day 27                   | #
# | 13866 팀 나누기           | #
# ---------------------------- #


if __name__ == "__main__":
    A, B, C, D = map(int, r_input().split())
    
    print(abs(D + A - C - B))

File: Baekjoon_Online_Judge/NO_13866/13866.c
#include <stdio.h>
#include <math.h>
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 07                   | 
 | 13866 팀 나누기           | 
 ---------------------------- 
*/

int main(){
    int A, B, C, D;

    scanf("%d %d %d %d", &A, &B, &C, &D);
    printf("%d", abs(D + A - C - B));
    return 0;
}